features/javascript/validate.feature

```diff
@@ -0,0 +1,159 @@
+@stable @javascript
+Feature: validate
+
+  Scenario: HTTP Request
+    Given you expect the following HTTP Request:
```

Comment:
ditto my note below, `I` is preferred over `you` (may apply to other places I've could overlook).